.editorconfig

```diff
@@ -52,7 +52,7 @@ indent_size = 2
 
 # Feature files
 [*.feature]
-end_of_line = lf
+end_of_line = crlf
```

Comment:
Just double checking if you meant to change this?

Reply:
Yeah was sick of it warning me about mixed line endings. So hopefully this will help.